src/components/Navbar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Navbar.css';
+
+function Navbar() {
+  return (
+    <div className="navbar-container">
+      <h1 className="navbar-logo">BookStore</h1>
+      <ul className="item-container">
+        <li><Link to="/" className="item">Books</Link></li>
```

Comment:
- [ ] Kindly check. This link should be leading to booklist.

Reply:
Hi @t-yanick and this is the same issue too, my instructions indicate to route to books and categories, and the booklist it's a component for books
![image](https://user-images.githubusercontent.com/114549019/213626363-bf087bc2-c5e9-474f-86fd-daadea6e17e9.png)

Reply:
- [ ] Sorry about the discrepancy between the Reviewers instructions and the project requirements. Kindly submit for re-review and the next reviewer will approve your PR.

Reply:
@t-yanick Thank you for checking, I already fixed the other issues regarding remove button and install command, thank you for the feedback =)